### PR TITLE
Add integration test for importing and exporting ONNX models

### PR DESCRIPTION
**Pull request by laurenyu**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by laurenyu** [comment]
there will be a corresponding a notebook :)

we do explicitly install onnx, so it's not entirely accurate to say we don't do anything special. I think we should make sure that the other libraries (keras-mxnet and onnx as of now) we include work as advertised

File: test/integration/local/test_onnx.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import os
+
+import numpy
+from sagemaker.mxnet import MXNet
+
+import local_mode
+from test.integration import NUM_MODEL_SERVER_WORKERS, RESOURCE_PATH
+
+
+def test_onnx_export_and_import(docker_image, sagemaker_local_session, local_instance_type,
```
Comment (icywang86rui) [inline test/integration/local/test_onnx.py]: Let's separate the import and export tests as discussed.

File: test/conftest.py [hunk 2]
```diff
@@ -35,7 +35,7 @@
 def pytest_addoption(parser):
     parser.addoption('--docker-base-name', default='preprod-mxnet')
     parser.addoption('--region', default='us-west-2')
-    parser.addoption('--framework-version', default='1.2.1')
+    parser.addoption('--framework-version', default='1.3.0')
```
Comment (icywang86rui) [inline test/conftest.py]: The tests would fail with all the older version now. Should we mention it in README?

Comment (laurenyu) [reply test/conftest.py]: not just the tests, but everything. I'll update the README when I remove the old Dockerfiles?

File: test/integration/__init__.py [hunk 3]
```diff
@@ -16,12 +16,10 @@
...
-    os.path.join('model', 'model-symbol.json'),
-    os.path.join('model', 'model-shapes.json'),
-    os.path.join('model', 'model-0000.params'),
-]
+MODEL_SUCCESS_FILES = {
+    'output': ['success'],
+    'model': ['model-symbol.json', 'model-shapes.json', 'model-0000.params'],
+}
 
 # Workaround for the intermittent worker timeout errors
```
[the remark this answers is not in the document]
Comment (laurenyu) [reply test/integration/__init__.py]: sure

File: test/conftest.py (hunk 2 above)
Comment (icywang86rui) [reply test/conftest.py]: that's fine with me.

**Pull request merged**